mesa/space.py

```diff
@@ -643,7 +644,7 @@ def iter_cell_list_contents(
         )
 
 
-class HexGrid(Grid):
+class _HexGrid(_Grid):
```

Comment:
This will break people's code, and should be deferred to 2.0.

Reply:
We probably have enough breaking changes for 2.0. Maybe the next release should be 2.0? Let's discuss on Jan 7th meeting.

Reply:
Regardless, the PR should contain commits that have the same theme. Implementing `SingleHexGrid`and `MultiHexGrid` should be done in a separate PR. This will screw with the changelog otherwise.

Reply:
right, I made HexGrid inherit from SingleGrid so that it place agents correctly